src/Test/Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -31,6 +32,7 @@ public abstract class DiagnosticAnalyzerTestBase
         private static readonly MetadataReference s_immutableCollectionsReference = MetadataReference.CreateFromAssemblyInternal(typeof(ImmutableArray<int>).Assembly);
         private static readonly MetadataReference s_systemDiagnosticsDebugReference = MetadataReference.CreateFromAssemblyInternal(typeof(System.Diagnostics.Debug).Assembly);
         private static readonly MetadataReference s_systemDataReference = MetadataReference.CreateFromAssemblyInternal(typeof(System.Data.DataSet).Assembly);
+        private static readonly MetadataReference s_SystemRuntimeSerializationReference = MetadataReference.CreateFromAssemblyInternal(typeof(DataMemberAttribute).Assembly);
```

Comment:
Instead of adding a reference, the test could just define the attribute - since we are only checking by name that would succeed. It's nicer to keep the set of references in the test framework to a minimum.